### ISSUE TITLE
Login via CAS UK problem

### ISSUE THREAD

**Issue by Lukas-Riedel** [ReCodEx/android-app#1]
I have a problem in the android app while logging in via CAS UK. Everytime I try to log in using my UKCO and PW, I get a "Password incorrect" (or sth. like that) error. I'm convinced the password I'm entering is correct.

Btw. I have created account in ReCodEx in the web app using CAS UK only.

**Comment by SemaiCZE** [comment]
Yes, we know. It's a while since we wrote the app and several things have changed. One is ReCodEx domain and second is a CAS authentication - now you don't give us your password to lookup your info in CAS LDAP, but you log in CAS web and we get the info from there. This needs to be somehow implemented in the Android app too, but now we have a lot of things to fix in regular web application, so this needs to wait. However, if anyone want to work on this (maybe as project for Developing Applications for Mobile Devices - NPRG056), we'd support him.
